### PR TITLE
LPS-102896 Remove unused imports

### PR DESCRIPTION
**Pull request by boton**

### CODE REVIEW

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_item_selector_preview.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.item-selector-preview {
```
Comment (boton) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_item_selector_preview.scss]: Maybe use atlas variables 🤔

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Carousel.es.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (boton) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Carousel.es.js]: Check proptypes is a good idea or remove unused import

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Header.es.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayNavigationBar from '@clayui/navigation-bar';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Header = ({
+	handleClickClose,
+	handleClickDone,
+	handleClickEdit,
```
Comment (boton) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/Header.es.js]: Check handleClickEdit proptype

File: modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/ItemSelectorPreview.es.js [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+	componentWillUnmount() {
+		document.documentElement.removeEventListener(
+			'keydown',
+			this.handleOnKeyDown.bind(this)
+		);
+	}
+
+	componentDidUpdate() {
+		const sidenavToggle = this.refs.test;
```
Comment (boton) [inline modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/ItemSelectorPreview.es.js]: Where is the ref `test`?

Comment (boton) [reply modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/ItemSelectorPreview.es.js]: Why this in componentDidUpdate, not in componentDidMount.

**Comment by ambrinchaudhary** [comment]
Just started reviewing :)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/ambrinchaudhary/liferay-portal/pull/258. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/a20eb24e42c6~2...a20eb24e42c6)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79345. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/a20eb24e42c6~2...a20eb24e42c6)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79348. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/bec1c2b8eadd~2...bec1c2b8eadd)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79349. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/4635b7fe0bfe~2...4635b7fe0bfe)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/4lejandrito/liferay-portal/pull/332. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/600b9244feb5~2...600b9244feb5)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79350. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/8e6cbac4cca3~2...8e6cbac4cca3)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79390. [See changes here.](https://github.com/ambrinchaudhary/liferay-portal/compare/3ecf143f3da8~2...3ecf143f3da8)

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79392.

**Comment by ambrinchaudhary** [comment]
Pull request submitted to https://github.com/boton/liferay-portal/pull/91.

**Pull request closed without merging**